Comp2023/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -538,6 +542,70 @@ public void drive(Translation2d translation, double rotation, boolean fieldRelat
     }
   }
 
+  public void driveBalance( )
+  {
+    pitch = m_pigeon.getUnadjustedPitch( ).getWPIRotation2d( ).getDegrees( );
+    DataLogManager.log("Balance Off By: " + pitch);
+
+    if (Math.abs(pitch) > 5)
```

Comment:
It's a good practice to use floating point numbers when comparing them to help the reader. So 5.0 is better than 5 here.